d4s2_api/models.py

```diff
@@ -395,6 +396,10 @@ class Meta:
         unique_together = ('endpoint', 'name')
 
 
+class S3ObjectManifest(models.Model):
+    content = JSONField(help_text='JSON array of object metadata from bucket at time of sending bucket')
```

Comment:
This is working nicely, I was able to write queries in python to walk through the internal JSON structure in postgres.